src/pip/_internal/utils/misc.py

```diff
@@ -104,6 +105,17 @@ def cast(typ, val):
     logger.debug('lzma module is not available')
 
 
+def get_pip_version():
```

Comment:
nit: It'll be nice to move the introduction of this function, into a separate commit.